app/networkapi/settings.py

```diff
@@ -131,15 +146,19 @@
         ],
         'APP_DIRS': True,
         'OPTIONS': {
-            'context_processors': [
+            'context_processors': list(filter(None, [
+                'social_django.context_processors.backends'
+                if SOCIAL_SIGNIN else None,
+                'social_django.context_processors.login_redirect'
+                if SOCIAL_SIGNIN else None,
```

Comment:
this was hard to parse in my head - I'm guessing it's on a new line because of length... Would some extra indendation make it easier to read, while still keeping lint happy?